can/interfaces/pcan/pcan.py

```diff
@@ -477,6 +477,31 @@ def state(self, new_state):
             )
 
 
+    @staticmethod
+    def _detect_available_configs():
+        libraryHandle = PCANBasic()
+        channels = []
+        interfaces = []
+        for i in range(16):
+            interfaces.append({'id': TPCANHandle(PCAN_PCIBUS1.value + i), 'name': 'PCAN_PCIBUS'+str(i+1)})
+        for i in range(16):
+            interfaces.append({'id': TPCANHandle(PCAN_USBBUS1.value + i), 'name': 'PCAN_USBBUS'+str(i+1)})
+        for i in range(2):
+            interfaces.append({'id': TPCANHandle(PCAN_PCCBUS1.value + i), 'name': 'PCAN_PCCBUS'+str(i+1)})
+        for i in range(16):
+            interfaces.append({'id': TPCANHandle(PCAN_LANBUS1.value + i), 'name': 'PCAN_LANBUS'+str(i+1)})
+        status = TPCANStatus(0)
+        for i in interfaces:
+            error, value = libraryHandle.GetValue(i['id'], PCAN_CHANNEL_CONDITION)
+            if error != PCAN_ERROR_OK or value != PCAN_CHANNEL_AVAILABLE:
+                continue
+            hasFd = False
+            error, value = libraryHandle.GetValue(i['id'], PCAN_CHANNEL_FEATURES)
+            if error == PCAN_ERROR_OK:
+                hasFd = bool(value & FEATURE_FD_CAPABLE)
+            channels.append({'interface': 'pcan', 'channel': i['name'], 'supportsFd': hasFd})
```

Comment:
This should probably be like that, to be consistent with the naming scheme:

```suggestion
            channels.append({'interface': 'pcan', 'channel': i['name'], 'supports_fd': hasFd})
```